store/sqlstore/post_store.go

```diff
@@ -1236,6 +1237,28 @@ func (s *SqlPostStore) search(teamId string, userId string, params *model.Search
 		searchClause := fmt.Sprintf("AND to_tsvector('english', %s) @@  to_tsquery('english', :Terms)", searchType)
 		searchQuery = strings.Replace(searchQuery, "SEARCH_CLAUSE", searchClause, 1)
 	} else if s.DriverName() == model.DATABASE_DRIVER_MYSQL {
+		if searchType == "Message" {
+			stopWords, err := findStopWordsMysql(s)
```

Comment:
I agree with your point. This should come from a static list. I'd put that list in a file called `stop_word.go` in `searchlayer` folder/package